central/networkgraph/service/service_impl.go

```diff
@@ -540,6 +540,10 @@ func (s *serviceImpl) addDeploymentFlowsToGraph(
 	// Aggregate all external flows by node names to control the number of external nodes.
 	flows = aggregator.NewDuplicateNameExtSrcConnAggregator().Aggregate(flows)
 	missingInfoFlows = aggregator.NewDuplicateNameExtSrcConnAggregator().Aggregate(missingInfoFlows)
+
+	flows = aggregator.NewAllAggregator().Aggregate(flows)
+	missingInfoFlows = aggregator.NewAllAggregator().Aggregate(missingInfoFlows)
```

Comment:
Based on the description in this PR, do I understand correctly that this will essentially do nothing unless we've anonymised external IP flows to the internet?

If so, we need a comment here describing why that's the case

Reply:
It is possible that the other aggregators are not correctly deduplicating connections. This solution is more resilient than trying to fix an aggregator. Currently I don't see problems with the other aggregators, but it is possible that there are or will be in the future.